CMakeLists.txt

```diff
@@ -2,12 +2,12 @@ cmake_minimum_required(VERSION 3.1...3.26)
 
 project(fsmlang VERSION 1.0 LANGUAGES C CXX)
 
-set(CMAKE_C_STANDARD 23)
+set(CMAKE_C_STANDARD 99)
```

Comment:
Why downgraded here?

Reply:
Inadvertent - my installation needs it (can't get wsl linux to upgrade); I will restore.